src/Psl/Type/Internal/MixedDictType.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Type\Internal;
+
+use Psl\Type;
+use Psl\Type\Exception\AssertException;
+use Psl\Type\Exception\CoercionException;
+
+/**
+ * @extends Type\Type<array<array-key, mixed>>
+ *
+ * @internal
+ */
+final class MixedDictType extends Type\Type
+{
+    /**
+     * @throws CoercionException
+     *
+     * @return array<array-key, mixed>
+     */
+    public function coerce(mixed $value): array
+    {
+        if (! is_iterable($value)) {
```

Comment:
I just noticed the PHP-cs fixed did not pick up on this not being imported in my dev, or in CI.